eval/eval.go

```diff
@@ -327,7 +327,7 @@ func evalIndexExpression(left, index object.Object) object.Object {
 		if idx < 0 || idx >= int64(len(str)) {
 			return object.NULL
 		}
-		return object.Integer{Value: int64(str[idx])}
+		return object.Integer{Value: int64(str[idx])} //nolint:gosec // https://github.com/securego/gosec/issues/1185
```

Comment:
A comment explaining the problem would be appreciated. Yes, there is a link, and it's great. but someone reading needs tpneedo open the link. People are lazy. I can tell I'm lazy 😬

Reply:
strings[idx] are bytes (unsigned uint8) and the linter thinks that 0-255 value can't fit into a int64 which is completely stupid

Reply:
the link to that bug is enough. and hopefully it get fixed, though it pisses me off golanglint-ci action pin doesn't pin the linters themselves and stuff start breaking mid PRs </rant>